Doc/library/dataclasses.rst

```diff
@@ -51,24 +51,24 @@ Module contents
    This function is a :term:`decorator` that is used to add generated
    :term:`special methods <special method>` to classes, as described below.
 
-   The ``@dataclass`` decorator examines the class to find
+   The :func:`@dataclass <dataclass>` decorator examines the class to find
    ``field``\s.  A ``field`` is defined as a class variable that has a
    :term:`type annotation <variable annotation>`.  With two
-   exceptions described below, nothing in ``@dataclass``
+   exceptions described below, nothing in :func:`@dataclass <dataclass>`
```

Comment:
I'd actually leave this one as-is, so as to avoid linking to the same thing twice in one paragraph (same for a few others in this PR)

Reply:
Thanks, I'll address that later today.

Reply:
I believe 08ea16b9591a604d1aa7988883145cd1ed6724cf should address this remark.

Reply:
it makes me happy but sphinx doesn't like it

Reply:
Yeah; it's really strange. `make check` worked locally; I think I'll revert to double backticks, like I initially suggested for Hugo earlier in this PR.

Reply:
Reverted in c41d8c46036f4c71845fa608a2e3e54622214bba. We can figure this out later.